packages/terra-form/docs/Field.md

```diff
@@ -10,6 +10,7 @@ import Field from 'terra-form/lib/Fieldset';
 
 <Field
   label="Do you have any Children?"
+  labelAttrs={{ className: 'healtheintent-application' }}
   error="This field is required"
   help="Families are eligible for family package plans"
   required
```

Comment:
I prefer options 1 and 3 if we were to need to scale this out with more preference on option 1.

styled-components has a [`.attrs` method](https://www.styled-components.com/docs/api#attrs) which accepts an object of attributes to add to an element. We could pass the object from things like `labelAttrs` directly to that.

Adding attributes directly to components like `<Label className="healtheintent-application">` should be an issue either.